src/features/todos.ts

```diff
@@ -1,9 +1,24 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type LoadTodos = { type: 'todos/LOAD', payload: Todo[] };
 
-const todosReducer = (): Todo[] => {
-  return [];
+type Action = LoadTodos;
+
+const loadTodos = (todos: Todo[]): Action => ({
+  type: 'todos/LOAD',
+  payload: todos,
+});
+
+export const actions = { loadTodos };
+
+const todosReducer = (todos: Todo[] = [] || null, action: Action) => {
+  switch (action.type) {
+    case 'todos/LOAD':
+      return [...todos, ...action.payload];
```

Comment:
The 'todos/LOAD' case is currently returning a new array that is a combination of the old todos and the new ones. If you're intending to replace the old todos with the new ones, then the return statement should be 'return action.payload;' instead.